unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/FriendsHUDController.cs

```diff
@@ -344,26 +348,6 @@ private void HandleFriendshipUpdated(string userId, FriendshipAction friendshipA
                 View.Set(userId, approved);
                 userProfile.OnUpdate += HandleFriendProfileUpdated;
                 break;
-            case FriendshipAction.REQUESTED_FROM:
```

Comment:
why cases `FriendshipAction.REQUESTED_FROM` & `FriendshipAction.REQUESTED_TO` has been removed?

Reply:
It is because this logic has been moved directly to `AddFriendRequests(...)` that is called after `await friendsController.GetFriendRequests(...)` and process the results. Now the friend requests list is filled using these results directly, instead of doing it managing the friendship status events.